v.30: pass config file to meilisearch (#1991)
meilisearch/documentation: learn/getting_started/quick_start.md

**Instance options affect your entire Meilisearch instance**, not just a single index.

> ### Configuration file

Reviewer: The config file only accepts instance options, right? If so, I think it's better to have it as subheading of `### Instance options` instead of a self-sufficient section like `### Index settings`.

```suggestion
#### Configuration file
```

Author: It is a subheading of `Instance options` 👀

Reviewer: Oh, but aren't both of them on the same heading level?

`### Instance options` on L#276

`### Configuration file` on L#282

Both are level three (`###`), but for configuration file to be a subheading it would have to be level four (`####`), no?

Author: I get it now! Yes, you're right 🙏

> Instead of providing command-line options or environment variables, you can use a `.toml` [configuration file](/learn/configuration/instance_options.md#configuration-file). Once downloaded, move it to your working directory and it will be loaded at launch. Like instance options, it affects your entire instance.

Reviewer 2: It's not necessary to download a premade configuration file, since users can also easily construct their own based on the formatting rules we provide.

```suggestion
As an alternative to providing command-line options or environment variables at launch, you can configure your Meilisearch instance using a [configuration file](/learn/configuration/instance_options.md#configuration-file) in `.toml` format. If a configuration file is present at the root of your working directory, it will be loaded at launch.
```

> Instead of providing command-line options or environment variables, you can provide a `.toml` [configuration file](/learn/configuration/instance_options.md#configuration-file). Once downloaded, move it to your working directory and it will be loaded at launch. Like instance options, it affects your entire instance.

Reviewer: Trying to avoid repeating the same verb twice in quick succession.

```suggestion
Instead of providing command-line options or environment variables, you can use a `.toml` [configuration file](/learn/configuration/instance_options.md#configuration-file). Once downloaded, move it to your working directory and it will be loaded at launch. Like instance options, it affects your entire instance.
```
